packaging/rpm-ostree.spec

```diff
@@ -241,6 +241,7 @@ $PYTHON autofiles.py > files \
   '%{_datadir}/dbus-1/system.d/*' \
   '%{_sysconfdir}/rpm-ostreed.conf' \
   '%{_prefix}/lib/systemd/system/*' \
+  '%{_prefix}/lib/kernel/install.d/*' \
```

Comment:
looks like this is wrong:

```
[2025-01-10T22:15:10.770Z] error: Installed (but unpackaged) file(s) found:
[2025-01-10T22:15:10.770Z]    /lib/kernel/install.d/05-rpmostree.install
```

Reply:
It's because it was installed to `/lib`, not `/usr/lib` (it's a symlink in the real system, but not in the RPM). IMO we should always prefer writing to `/usr/lib` so change the Makefile.

Reply:
I am not sure why this resolves the container build failing and I double checked and the file is in the correct place. Will do more manual testing before merging this. Because is weird that now even the container-integration tests has no issue removing the old kernel.